src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java

```diff
@@ -356,6 +357,12 @@
     )
     private List<String> transformationAndOcrWarnings = new ArrayList<>();
 
+    @CCD(
+        label = "Retired Digital Notice of Proceedings?",
+        access = {CaseworkerAndSuperUserAccess.class}
```

Comment:
you can remove the access here now

Reply:
Removed.
https://github.com/hmcts/nfdiv-case-api/pull/1433/commits/b22c4d5c85c97bfc2a7cb862b06279baaadfd4e3